src/hotspot/share/opto/compile.cpp

```diff
@@ -1996,6 +1996,27 @@ bool Compile::optimize_loops(PhaseIterGVN& igvn, LoopOptsMode mode) {
   return true;
 }
 
+// execute posponed transformations
+void Compile::post_optimize_loops_igvn(PhaseIterGVN& igvn) {
```

Comment:
Preferred way is to register nodes for processing using `Compile::record_for_post_loop_opts_igvn()`.
That way you avoid additional pass over the graph.